src/main/webapp/isolate/request/request-form.html

```diff
@@ -0,0 +1,20 @@
+<!--
+  ~ Copyright 2020 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<button onclick="addTicket()">Create Ticket</button>
```

Comment:
Is a ticket intended as a request for a volunteer to perform a task?

Reply:
Yes, that's it exactly. Each ticket would either be an item they wanted retrieved from a shop/pharmacy, or would be a task that the isolate wanted performed e.g. walking the dogs.